### ISSUE TITLE
Release Python 3.11 wheels

### ISSUE THREAD

**Issue by hawkinsp** [google/jax#12807]
Python 3.11 will be released on October 24 (https://www.python.org/downloads/release/python-3110rc2).

The ABI is stable, so we are free to build and release wheels using the rc2 release. JAX already works and the tests pass. NumPy and SciPy have already released Python 3.11 wheels.

We should add Python 3.11 wheels to our jaxlib release process.

**Comment by yashk2810** [comment]
With that, can we deprecate 3.7?

**Comment by hawkinsp** [comment]
Colab upgrading past 3.7 blocks deprecating 3.7. But hopefully that won't be long. We shouldn't wait for that before adding 3.11 wheels, though.

**Comment by jakevdp** [comment]
I think we need to continue supporting Python 3.7 until Colab migrates. 5 wheels is a lot, but it will be temporary.

**Comment by wookayin** [comment]
Python 3.11 has finally landed (released 10/24), greatly looking forward to jaxlib release for python 3.11!

**Comment by hawkinsp** [comment]
Yes, everything is ready for 3.11 and has been for some time. Feel free to release the most recent jaxlib for 3.11.

**Comment by yashk2810** [comment]
jax and jaxlib 0.3.24 have been released with python 3.11 support
